knowledge-base/guides/custom-lightning-address.mdx

```diff
@@ -130,7 +130,7 @@ Navigate to the Doc Portal. Add this to your payload AFTER changing the `identif
 
 ```json
 {
-  "identifier": "sats@santos.lol",
```

Comment:
Pretty sure this has to be `identifier` and not `internalId` those are different things.